capitalize_decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'nameable'
+
+# create CapitalizeDecorator class
+class CapitalizeDecorator < Decorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.capitalize
```

Comment:
- [ ] @sheezu900 Great work on the capitalize decorator class, however, this method won't capitalize our word, the nameable variable contains an object that wee need to find the word we need to capitalize eg ``@nameable.correct_name.capitalize``